### DISCUSSION TITLE
About support for RokidAirProPlus devices

### DISCUSSION THREAD

**Discussion by LiamLsc** [GingerXR/products#2] [Q&A]
I hope to be able to adapt to the Rokid Air Pro Plus, because this device requires external third-party Android devices, such as Android phones, resulting in the need to wear glasses every time during development to view the perspective inside the device. Can I transfer the images inside the glasses to a PC

**Comment by GingerXR** [comment]
Hi LiamLsc,

I suggest to transfer the files using a cloud file system like OneDrive. 

We are working on adding wired support (Rokid Air plug directly into PC; not sure if it would 100% work for the pro). But would that solve your problem?
